packages/auth/src/tests/buildAccessPolicy.test.js

```diff
@@ -10,12 +10,12 @@ import {
   getTestDatabase,
   upsertDummyRecord,
   findOrCreateDummyRecord,
-  ModelRegistry,
+  getTestModels,
 } from '@tupaia/database';
 import { buildAccessPolicy } from '../buildAccessPolicy';
 
 describe('buildAccessPolicy', () => {
-  const models = new ModelRegistry(getTestDatabase());
+  const models = getTestModels();
```

Comment:
This was a common pattern, so I created a util that instantiates a new model registry with the test db

Reply:
Great!!